src/ConfigAndContainerTrait.php

```diff
@@ -34,6 +34,18 @@ private function getContainer(string $projectPath) : ContainerInterface
      */
     private function getConfig(string $projectPath) : array
     {
-        return $this->getContainer($projectPath)->get('config');
+        $config = $this->getContainer($projectPath)->get('config');
+
+        if (is_array($config)) {
+            return $config;
+        }
+
+        if (! $config instanceof \Traversable) {
+            throw new \InvalidArgumentException(
```

Comment:
it is a runtime condition, so RuntimeException.
I would also suggest to improve message like so:
```php
sprintf(
    '"config" service must be an array or instance of ArrayObject or Traversable, got %s',
    is_object($config) ? get_class($object) : gettype($object)
);
```